reference/api/documents.mdx

```diff
@@ -301,29 +392,79 @@ Delete one document based on its unique id.
 
 You can use this `taskUid` to get more details on [the status of the task](/reference/api/tasks#get-one-task).
 
+## Delete documents by filter
+
+<RouteHighlighter method="POST" route="/indexes/{index_uid}/documents/delete"/>
+
+Delete a selection of documents based on a filter.
+
+### Path parameters
+
+| Name              | Type   | Description                                                            |
+| :---------------- | :----- | :--------------------------------------------------------------------- |
+| **`index_uid`** * | String | [`uid`](/learn/core_concepts/indexes#index-uid) of the requested index |
+
+### Body
+
+A filter expression written as a string or array of array of strings for the documents to be deleted.
```

Comment:
"string or array of array of strings"

is there a repeated "array of" here?

Reply:
>is there a repeated "array of" here?

Yes. It's `string` or `array of array of strings`